ci/deploy.sh

```diff
@@ -28,41 +30,45 @@ gcloud docker -- push ${PHP_REPOSITORY}:latest;
 gcloud docker -- push ${NGINX_REPOSITORY}:latest;
 gcloud docker -- push ${VARNISH_REPOSITORY}:latest;
 
+
+echo "Installing or upgrading release"
 # Perform a rolling update if a release in the given namespace ever exist, create one otherwise.
 # Be aware that we have the static ip for the master branch but it belongs to you to care about others.
-helm upgrade --install --reset-values --wait --force --namespace=${BRANCH} --recreate-pods demo ./api/helm/api STATIC_IP \
+helm upgrade --install --reset-values --wait --force --namespace=${NAMESPACE} --recreate-pods ${RELEASE} ./api/helm/api ${STATIC_IP} \
     --set php.repository=${PHP_REPOSITORY} \
     --set nginx.repository=${NGINX_REPOSITORY} \
     --set varnish.repository=${VARNISH_REPOSITORY} \
     --set secret=${APP_SECRET} \
     --set postgresUser=${DATABASE_USER},postgresPassword="${DATABASE_PASSWORD}",postgresDatabase=${DATABASE_NAME} --set postgresql.persistence.enabled=true;
 
-kubectl exec -it $(kubectl --namespace=${BRANCH} get pods -l app=api-php -o jsonpath="{.items[0].metadata.name}") --namespace=${BRANCH} \
-    -- ash -c'export APP_ENV=dev && composer install -n && bin/console d:s:u --force --env=dev && bin/console hautelook:fixtures:load -n && APP_ENV=prod composer --no-dev install --classmap-authoritative && bin/console d:s:u --env=prod';
+echo "Waiting for api-php to be up and ready..."
+sleep 60
```

Comment:
Do we need this as we use `--wait` in `helm upgrade` ?

Reply:
Yes, cause the `--wait` doesn't wait for containers to be healthy, which is required to run `exec` just then. This process will be improved by https://github.com/api-platform/demo/issues/79#issuecomment-454903197